### ISSUE TITLE
Pass more options to --pull-request [Feature request]

### ISSUE THREAD

**Issue by philmd** [stefanha/git-publish#104]
`git-publish --pull-request` push a tag to the repository.
When this repository is hosted on Gitlab.com and has Continuous Integration enabled,
this triggers a new pipeline by default. Pipelines aren't free.
The option to not trigger pipelines is to use push using `--push-option=ci.skip` (see https://docs.gitlab.com/ee/user/project/push_options.html).

**Comment by stefanha** [comment]
Good idea. I think there should be an option since whether or not to run CI depends on the workflow.

Some people may want CI to run, especially if the maintainer merging the request wants to see the CI results.

Others will have already run CI on their branch before pushing it and need ``--push-option=ci.skip``.
